inst/templates/easydashboard.Rmd

```diff
@@ -0,0 +1,153 @@
+  out.width = "100%"
+)
+```
+
+```{r}
+# Get user-specified model data
+model <- params$model
+parameters_args <- c(list(model), params$parameters_args)
+performance_args <- c(list(model), params$performance_args)
+
+table_parameters <- do.call(parameters::parameters, parameters_args)
+table_performance <- do.call(performance::performance, performance_args)
+
+# modelbased
+int_terms <- find_interactions(model, component = "conditional", flatten = TRUE)
+con_terms <- find_variables(model)$conditional
+
+if (is.null(int_terms)) {
+  model_terms <- con_terms
+} else {
+  model_terms <- clean_names(int_terms)
+  int_terms <- unique(unlist(strsplit(clean_names(int_terms), ":", fixed = TRUE)))
+  model_terms <- c(model_terms, setdiff(con_terms, int_terms))
+}
+text_modelbased <- lapply(unique(model_terms), function(i) {
+  grid <- get_datagrid(model, at = i)
+  estimate_expectation(model, data = grid)
+})
+
+text_report <- report(model)
+```
+
+Model fit 
+=====================================  
+
+Column {data-width=700}
+-----------------------------------------------------------------------
+
+### Assumption checks
+
+```{r check-model, fig.height=10, fig.width=10}
+check_model(model)
+```
+
+Column {data-width=300}
+-----------------------------------------------------------------------
+
+### Indices of model fit
+
+```{r}
+print_md(table_performance, layout = "vertical", caption = NULL)
+```
+
+For interpretation of performance metrics, please refer to [this documentation](https://easystats.github.io/performance/reference/model_performance.html).
+
+Parameter estimates
+=====================================  
+
+Column {data-width=550}
+-----------------------------------------------------------------------
+
+### Plot
+
+```{r dot-whisker}
+plot(table_parameters)
+```
+
+Column {data-width=450}
+-----------------------------------------------------------------------
+
+### Tabular summary
+
+```{r}
+print_md(table_parameters, caption = NULL)
+```
+
+To find out more about table summary options, please refer to [this documentation](https://easystats.github.io/parameters/reference/model_parameters.html).
+
+
+Predicted Values
+=====================================  
+
+Column {data-width=600}
+-----------------------------------------------------------------------
+
+### Plot
+
+```{r expected-values, fig.height=10, fig.width=10}
+ggplot2::theme_set(theme_modern())
+# all_plots <- lapply(text_modelbased, function(i) {
+#   out <- do.call(visualisation_recipe, c(list(i), modelbased_args))
+#   plot(out) + ggplot2::ggtitle("")
+# })
+all_plots <- lapply(text_modelbased, function(i) {
+  out <- visualisation_recipe(i, show_data = "none")
+  plot(out) + ggplot2::ggtitle("")
+})
+see::plots(all_plots, n_columns = round(sqrt(length(text_modelbased))))
+```
+
+Column {data-width=400}
+-----------------------------------------------------------------------
+
+### Tabular summary
+
+```{r}
+lapply(text_modelbased, function(i) {
+  print_md(i)
+})
+```
```

Comment:
Now someone needs to find out how to create markdown tables from objects stored in a list.

Reply:
Either:

(cf. `purrr::walk`)

```r
hidden_lapply <- function(X, FUN, ...) {
  lapply(X, FUN, ...)
  invisible(X)
}
hidden_lapply(text_modelbased, print_md)
```

Or:
```
for (i in text_modelbased) print_md(i)
```